src/modules/settings/components/settings/BlacklistKeywords.jsx

```diff
@@ -47,9 +48,13 @@ function BlacklistKeywords() {
   );
 }
 
-export default registerComponent(BlacklistKeywords, {
-  settingId: SettingIds.BLACKLIST_KEYWORDS,
-  name: 'Blacklist Keywords',
-  category: CategoryTypes.CHAT,
-  keywords: ['black', 'list', 'keywords', 'banned', 'remove', 'hide'],
-});
+loadModuleForPlatforms([
```

Comment:
since a majority of settings apply specifically to Twitch, does it make sense to have twitch-specific settings in a twitch folder instead of having this boilerplate for each?